ngi_pipeline/engines/piper_ngi/workflows.py

```diff
@@ -122,10 +122,19 @@ def workflow_dna_variantcalling(qscripts_dir_path, setup_xml_path, global_config
     job_walltime = slurm_time_to_seconds(config.get("slurm", {}).get("time") or "4-00:00:00")
     if output_dir:
         cl_string += " --output_directory {output_dir}"
+    job_native_args = config.get("piper", {}).get("jobNative")
+    if job_native_args:
+        # This should be a list
+        if type(job_native_args) is not list:
+            LOG.warn('jobNative arguments in config file specified in invalid '
+                     'format; should be list. Ignoring these parameters.')
+        else:
+            cl_string += " -jobNative {}".format(" ".join(job_native_args))
     if exec_mode == "sbatch":
         # Execute from within an sbatch file (run jobs on the local node)
-        num_threads = int(config.get("piper", {}).get("threads") or 8)
-        job_runner = "Shell"
+        num_threads = int(config.get("piper", {}).get("threads") or 16)
+        job_runner = config.get("piper", {}).get("shell_jobrunner") or \
+                     "ParallelShell --super_charge --ways_to_split 4 --job_scatter_gather_directory $SNIC_TMP/scatter_gather"
```

Comment:
Has anyone tested this? I was getting some samtools errors when I ran the tests but Johan ran with the same cl and it worked fine so we should probably triple-check. Just a heads up!